### PR TITLE
#151168138 czars should be able to  draws cards by clicking

### PR DESCRIPTION
**Pull request by gbengaoyetade**
#### What does this PR do?
Enable Czars to draw cards before each game starts
#### Description of Task to be completed?
As a CZAR I should be prompted to click to select from the deck of black cards the next question card.
#### How should this be manually tested?
Start a new game
#### Any background context you want to provide?
None
#### What are the relevant pivotal tracker stories?
#151168138
#### Screenshots (if appropriate)
screen shot 2017-10-12 at 8 15 55 am

#### Questions:
none

### CODE REVIEW

**Review by nebanat — APPROVED** [review]

**Review by ibonly — APPROVED** [review]
Get third approval before you merge!!!

**Review by segunolalive — APPROVED** [review]

**Pull request merged**
